src/components/Person/Person.jsx

```diff
@@ -1 +1,39 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+const MAN_GENDER = 'm';
+
+function getMarriedInfo(status, gender, companion) {
```

Comment:
It is worth choosing more meaningful names for arguments so that they explain what is happening in the code.
```suggestion
function getMarriedInfo(isMarried, gender, partnerName) {
```